dotCMS/src/main/java/com/dotcms/aspects/DelegateMethodInvocation.java

```diff
@@ -27,7 +25,7 @@
      * Proceeds to the next interceptor in the chain
      * @return Object usually the method
      */
-    Object proceed();
+    Object proceed() throws Throwable;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)